components/driver/include/driver/rmt.h

```diff
@@ -666,7 +668,7 @@ esp_err_t rmt_fill_tx_items(rmt_channel_t channel, const rmt_item32_t* item, uin
  *     - ESP_ERR_INVALID_ARG Parameter error
  *     - ESP_OK Success
  */
-esp_err_t rmt_driver_install(rmt_channel_t channel, size_t rx_buf_size, int intr_alloc_flags);
+esp_err_t rmt_driver_install(rmt_channel_t channel, size_t rx_buf_size,int min_items_count, int intr_alloc_flags);
```

Comment:
Unfortunately this change would break code written for previous 3.x IDF versions, so we can't take it as is.

Reply:
RF receivers (like this one)
<img src = "https://image.ibb.co/bK73dV/descarga.jpg">
send many short pulses in the absence of carrier this causes "RMT RX BUFFER FULL" log spam and waking up the application task unnecessarily wasting CPU time.
I can modify the code to not break backward compatibility if you want